internal/ecma335encoding/ecma335encoding.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package ecma335encoding
```

Comment:
I don't like the name of this package. What about just `ecma335`? It could then be used for other ecma335 internal things. This pattern is followed extensively in the Go source code, you won't find a `dwarf/encoding` or `dwarfencoding` package, just `dwarf`.

Reply:
Agreed, I wasn't totally sure that I was going to end up with this still split off into its own package and hadn't really properly considered the name yet.